numpy/random/_generator.pyx

```diff
@@ -338,6 +338,16 @@ cdef class Generator:
         out : ndarray or scalar
             Drawn samples from the parameterized beta distribution.
 
+        Examples
+        --------
+        >>> np.random.beta(0.5, 0.5, 3) // Scalar values for \\alpha and \\beta, size = 3, results in list with 3 samples from beta distribution
```

Comment:
We don't use C++ `//` style comments. Put the text above the code snippet. Please see the other examples for how to format the mix of text and code.

Reply:
The text also needs to be explicitly wrapped to <= 75 characters.